dist/js/eslint.test.js

```diff
@@ -0,0 +1,11 @@
+function eslintFunctionTest() {
+  return true;
+}
+
+eslintTest();
+var eslintObjectTest = function () {
```

Comment:
Thanks for the review. eslint.test.js file contains on purpose es6 violations so Hound CI would comment on errors.